reactive/easyrsa.py

```diff
@@ -334,6 +336,11 @@ def upgrade():
     clear_flag('easyrsa.configured')
 
 
+@hook('pre-series-upgrade')
+def pre_series_upgrade():
+    status_set('blocked', 'Series upgrade in progress')
```

Comment:
Seems like this status could easily get overwritten by other handlers. I recommend setting it in an atexit handler or refactoring the charm to use layer-status.